hubtools/src/lib.rs

```diff
@@ -858,6 +980,7 @@ impl RawHubrisArchive {
         root_certs: Vec<Certificate>,
         signing_certs: Vec<Certificate>,
         version: Option<&String>,
+        epoch: Option<&String>,
```

Comment:
This could be an `Option<u32>` here as well, presumably?

Reply:
done